src/view/ui/PropertyLine.svelte

```diff
@@ -10,7 +10,8 @@
     export let monster: Monster;
     export let item: PropertyItem;
 
-    let property = stringify(monster[item.properties[0]]);
+    let property = monster[item.properties[0]] ?? "";
+    property = property.toString();
```

Comment:
The `stringify` method was used to recurse through properties that can't be directly turned into strings (arrays, objects, etc - common from 5e.tools data).